src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.cs

```diff
@@ -160,6 +160,7 @@ public void CopyTo(System.Formats.Asn1.AsnWriter destination) { }
         public bool EncodedValueEquals(System.ReadOnlySpan<byte> other) { throw null; }
 #if NET9_0_OR_GREATER
         public TReturn Encode<TReturn>(System.Func<System.ReadOnlySpan<byte>, TReturn> encodeCallback) { throw null; }
+        public void Encode<TState>(TState state, System.Action<TState, System.ReadOnlySpan<byte>> encodeCallback) { }
```

Comment:
Missing the generic (un)constraint?

Reply:
... Huh. Seems like `GenerateReferenceAssemblySource` does not generate APIs with with the `allows ref struct` constraint (and removes them from existing methods?)

Filed a bug. https://github.com/dotnet/sdk/issues/47113